code/controllers/subsystem/hijack.dm

```diff
@@ -83,10 +83,17 @@ SUBSYSTEM_DEF(hijack)
 	/// If ARES has announced the 50% point yet for SD
 	var/ares_sd_announced = FALSE
 
+/// to fallow what stage fuel pump are in.
+var/global/fuelpump_stage = 0
```

Comment:
var/static/ is the norm I think, not var/global/

Reply:
Also, /global/ and GLOBAL_VAR_INIT() are not the same thing, pretty sure if you're just trying to create a "global" variable  GLOBAL_VAR_INIT(fuelpump_stage, 0) would be the right approach.